docs/mkdocs/documentation/day0_day1_handover.md

```diff
@@ -0,0 +1,83 @@
+# Managing Day0/Day1 kmods with KMM
+
+Some kmods might be installed without KMM. In order to enhance KMM's UX we
+could, in some cases, help customers to transition the lifecycle management
+of they kmods to KMM.
+
+## Definitions
+
+### Day 0
+
+The most basic kmods that are required for a node to become “Ready” in the cluster
+
+**Examples**
+* A storage driver that is required in order to mount the rootFS as part of the boot process.
+  Vendors will usually work closely with the RHEL team to make those drivers
+  in-tree so we won’t worry about them too much here.
+* A network driver that is required for the machine to access machine-config-server
+  on the bootstrap node to pull the ignition and join the cluster
+
+### Day 1
+
+Kmods that are not required for a node to become “Ready” in the cluster but are
+preferred to be installed/updated before the node is becoming “Ready”.
+
+**Examples**
+* An OOT network driver that replaces an outdated in-tree driver to exploit the full potential of the NIC.
+
+### Day2
+
+Kmods that can be dynamically loaded to the kernel or removed from it without
+interfering with the cluster infrastructure (such as connectivity).
```

Comment:
Already the case with day 1, no?

Reply:
Not exactly. Let's take the day1 use case that @yevgeny-shnaidman has worked on - an up-do-date OOT network driver that replaces an outdated in-tree network driver - it cannot be dynamically unloaded from the node without breaking the node's connection.